app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class UserMailer < ApplicationMailer
+  default from: 'agustin.bajo@wolox.com.ar'
+
+  def welcome_email(rent)
+    @rent = rent
+    @book = rent.book
+    mail(to: rent.user.email, subject: 'Welcome to my kickoff proyect ') do |format|
```

Comment:
Use I18n also on the subject, specially since you are using it already on the email view: https://guides.rubyonrails.org/i18n.html

Reply:
Done.